packages/lisk-transactions/src/utils/sign_and_verify.ts

```diff
@@ -131,15 +127,15 @@ export const verifyMultisignatures = (
 	// Transaction is waiting for more signatures
 	if (signatures.length < minimumValidations) {
 		return {
-			verified: false,
+			valid: false,
 			errors: [
 				new TransactionPendingError(`Missing signatures`, id, '.signatures'),
 			],
 		};
 	}
 
 	return {
-		verified:
+		valid:
 			verifiedSignatures.size >= minimumValidations &&
```

Comment:
The naming should be consistent. We should either use verifiedSignatures or validateSignatures and not both interchangingly.